neps/utils/cli.py

```diff
@@ -56,7 +70,49 @@ def init_config(args: argparse.Namespace) -> None:
     exist.
     """
     config_path = Path(args.config_path) if args.config_path else Path("run_config.yaml")
-    if not config_path.exists():
+
+    if args.state_machine:
```

Comment:
I saw this in the previous PR, probably better to not directly refer to it as a state machine. This is not something that is intuitive to the user in any way as they do not know how NePS runs internally. Not sure of better alternatives but I'm sure you can think of one ;)